doc/interop-test-descriptions.md

```diff
@@ -1152,7 +1152,8 @@ Procedures:
       }
     }
     ```
-2. After getting a response, client waits up to 5 seconds to receive a OOB load 
+2. After getting a response, client waits up to 10 seconds (or a total of 30s for the
```

Comment:
Text LGTM; please rewrap the lines though.

Reply:
what is the line length requirement? yapf is not complaining.